ts/components/RegionServicePlayground.tsx

```diff
@@ -0,0 +1,194 @@
+import { Body, Container, Content, Left, Text, View } from "native-base";
+import * as React from "react";
+import { Alert, ScrollView, StyleSheet, TextInput } from "react-native";
+import { heightPercentageToDP } from "react-native-responsive-screen";
+import WebView, { WebViewMessageEvent } from "react-native-webview";
+import I18n from "../i18n";
+import customVariables from "../theme/variables";
+import { getLocalePrimaryWithFallback } from "../utils/locale";
+import {
+  closeInjectedScript,
+  AVOID_ZOOM_JS,
+  APP_EVENT_HANDLER
+} from "../utils/webview";
+import ButtonDefaultOpacity from "./ButtonDefaultOpacity";
+import { Monospace } from "./core/typography/Monospace";
+import ActivityIndicator from "./ui/ActivityIndicator";
+import AppHeader from "./ui/AppHeader";
+import IconFont from "./ui/IconFont";
+
+type Props = {
+  onModalClose: () => void;
+};
+
+const styles = StyleSheet.create({
+  textInput: { padding: 1, borderWidth: 1, height: 30 },
+  contentPadding: { paddingHorizontal: customVariables.contentPadding },
+  webViewHeight: { height: heightPercentageToDP("50%") }
+});
+
+const RegionServiceWebViewPlayGround: React.FunctionComponent<Props> = (
+  props: Props
+) => {
+  const [navigationURI, setNavigationUri] = React.useState("");
+  const [webMessage, setWebMessage] = React.useState("");
+  const [text, setText] = React.useState("");
+  const [loading, setLoading] = React.useState(false);
+  const [success, setSuccess] = React.useState(false);
+  const [error, setError] = React.useState(false);
+
+  React.useEffect(() => {
+    // const cookie: Cookie = {
+    //   name: "test",
+    //   value: "test",
+    //   domain: "192.168.1.20", // the domain on which you need to add the cookie, in this case the IP on your machine
+    //   path: "/",
+    //   version: "1",
+    //   expires: "2021-05-30T12:30:00.00-05:00"
+    // };
+    /**
+     * the domain on which you need to add the cookie, in this case the IP on your machine
+     * this value and domain value on cookie MUST match
+     * this parameter is considered only by Android.
+     */
+    // CookieManager.set("http://192.168.1.20", cookie).catch(_ =>
+    //   showToast(
+    //     `Unable to execute cookie authentication for ${cookie.name}`,
+    //     "danger"
+    //   )
+    // );
+
+    if (navigationURI === "") {
```

Comment:
I guess also this could be removed.
It's fine that it starts without any url set